code/adam/python/solutions/lab3.py

```diff
@@ -0,0 +1,73 @@
+"""
+Adam
+Lab 3: Pick 6
+Date: Mar 15, 2023
+"""
+
+# Generate a list of 6 random numbers representing the winning tickets
+import random
+winner = []
+num = 6
+for x in range(6):
+    winner.append(random.randint(1,99))
+    winning_ticket = winner
+print(f"Winning tickets are {winning_ticket}")
+
+# Start your balance at 0
+balance = 0
+
+    
+
+
+# Loop 100,000 times, for each loop:
+entry = 0
+for x in range(100_000):
+    
+    if entry == 100_000:
```

Comment:
Since you are using a for loop, your loop will end on its own after 100,000 iterations.